client/src/main/java/com/vesoft/nebula/client/graph/net/SyncConnection.java

```diff
@@ -112,6 +135,57 @@ public void open(HostAddress address, int timeout)
         }
     }
 
+    /**
+     * create protocol for http2 with tls
+     */
+    private void getProtocolWithTlsHttp2() {
+        String url = "https://" + serverAddr.getHost() + ":" + serverAddr.getPort();
+        TrustManager trustManager;
+        if (SslUtil.getTrustManagers() == null || SslUtil.getTrustManagers().length == 0) {
+            trustManager = null;
+        } else {
+            trustManager = SslUtil.getTrustManagers()[0];
+        }
+        this.transport = new THttp2Client(url, sslSocketFactory, trustManager)
+                .setConnectTimeout(timeout)
```

Comment:
connect/read timeout is the same?

Reply:
yeah, In client 3.x, they always share the same value.